test/src/test/java/org/corfudb/integration/LogReplicationDynamicStreamIT.java

```diff
@@ -0,0 +1,520 @@
+import org.corfudb.runtime.collections.Table;
+import org.corfudb.runtime.collections.TableOptions;
+import org.corfudb.runtime.collections.TxnContext;
+import org.corfudb.test.SampleSchema.ValueFieldTagOne;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Collections;
+import java.util.UUID;
+import java.util.concurrent.CountDownLatch;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.corfudb.infrastructure.logreplication.replication.receive.LogReplicationMetadataManager.REPLICATION_STATUS_TABLE;
+import static org.corfudb.runtime.view.TableRegistry.CORFU_SYSTEM_NAMESPACE;
+
+/**
+ * This suite of tests validates the behavior of Log Replication
+ * when the set of streams to replicated is dynamically built by
+ * querying registry table instead of provided by static file.
+ */
+@Slf4j
+public class LogReplicationDynamicStreamIT extends LogReplicationAbstractIT {
+
+    private Table<Sample.StringKey, Sample.IntValue, Sample.Metadata> mapA;
+    private Table<Sample.StringKey, Sample.IntValue, Sample.Metadata> mapAStandby;
+    private final int numWrites = 2000;
+
+    /**
+     * Sets the plugin path before starting any test
+     */
+    @Before
+    public void setupPluginPath() throws Exception {
+        String nettyConfigDynamic = "src/test/resources/transport/nettyConfig.properties";
+        if(runProcess) {
+            File f = new File(nettyConfigDynamic);
+            this.pluginConfigFilePath = f.getAbsolutePath();
+        } else {
+            this.pluginConfigFilePath = nettyConfigDynamic;
+        }
+
+        // Initiate Source and Sink runtime and CorfuStore
+        setupActiveAndStandbyCorfu();
+
+        // Open replication status table to for verification purpose
+        corfuStoreActive.openTable(LogReplicationMetadataManager.NAMESPACE,
+                REPLICATION_STATUS_TABLE,
+                ReplicationStatusKey.class,
+                ReplicationStatusVal.class,
+                null,
+                TableOptions.fromProtoSchema(ReplicationStatusVal.class));
+
+        corfuStoreStandby.openTable(LogReplicationMetadataManager.NAMESPACE,
+                REPLICATION_STATUS_TABLE,
+                ReplicationStatusKey.class,
+                ReplicationStatusVal.class,
+                null,
+                TableOptions.fromProtoSchema(ReplicationStatusVal.class));
+    }
+
+    /*
+     * Helper methods section begin
+     */
+
+    private void openMapAOnActive() throws InvocationTargetException, NoSuchMethodException, IllegalAccessException {
+        // Open to StreamA on Source Site
+        mapA = corfuStoreActive.openTable(
+                NAMESPACE,
+                streamA,
+                StringKey.class,
+                IntValue.class,
+                Metadata.class,
+                TableOptions.fromProtoSchema(IntValueTag.class)
```

Comment:
The value schema is of type IntValue and the table options to be fetched from IntValueTag.  Although not causing functional issues, this is incorrect usage of the table open() method.  If you think about it, it means that the options for a table are specified in another protobuf.  The recommended way is to specify them in the value schema.  

If making further changes to the PR, consider changing the value schema to IntValueTag.